package.json

```diff
@@ -25,46 +26,40 @@
   },
   "homepage": "https://github.com/FACN4/StudentProject-WeCook#readme",
   "devDependencies": {
-    "@babel/core": "^7.0.1",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-react": "^7.0.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^9.0.0",
-    "babel-jest": "^23.6.0",
-    "babel-loader": "^8.0.0-beta.6",
-    "babel-plugin-styled-components": "^1.6.4",
-    "css-loader": "^1.0.0",
     "eslint": "^5.5.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.11.1",
     "eslint-plugin-react-redux": "^2.3.0",
-    "file-loader": "^2.0.0",
-    "html-webpack-plugin": "^3.2.0",
     "jest": "^23.5.0",
     "nodemon": "^1.18.4",
     "nyc": "^13.0.1",
     "react-hot-loader": "^4.3.6",
     "react-testing-library": "^5.0.1",
-    "style-loader": "^0.23.0",
     "supertest": "^3.3.0",
     "tap-spec": "^5.0.0",
     "tape": "^4.9.1",
-    "watch": "^1.0.2",
-    "webpack": "^4.17.2",
-    "webpack-cli": "^3.1.0",
     "webpack-dev-server": "^3.1.8"
   },
   "dependencies": {
+    "@babel/core": "^7.0.1",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
     "axios": "^0.18.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^9.0.0",
```

Comment:
also is this being used ? if yes isn't it a dev dep ?

Reply:
babel-core: Don't think it is being used, have removed
babel-eslint: yeah should be a dev dep, moving it